imt-proto.d.ts

```diff
@@ -129,6 +129,7 @@ declare namespace imtProto {
 
         initialize(done: (err?: Error) => void): void;
         finalize(done: (err?: Error) => void): void;
+        audit(payload): void;
```

Comment:
Could we please have at least a minimal type definition for the `payload` argument? 🙏  I'd say we at least expect it to be an object? Or is it completely abstract? Then we could at least mark it as `any`, just so we know what to expect 😁

Reply:
oops, the hazard of copy-pasting :sweat_smile:. We expect the payload to consist of the properties found [here](https://github.com/integromat/imt-app-runtime/blob/EMA-2358/lib/core/middleware/audit.js#L2) and `connection`, but I am unsure whether it's desired to define the specific properties on the proto level - if not then I can leave it as `any`?

Reply:
Got it, how about putting `Record<string, unknown>` there then? 🤔 As it will at least force the payload to be a string-keyed object, without actually enforcing anything else?